src/ansi-c/CMakeLists.txt

```diff
@@ -3,20 +3,57 @@ generic_flex(ansi_c)
 
 add_executable(converter library/converter.cpp)
 
+set(ansi_c_library_sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/cegis.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/float.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/locale.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/pthread_lib.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/string.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/windows.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/ctype.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/gcc.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/math.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/semaphore.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/strings.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/x86_assembler.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/err.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/getopt.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/netdb.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/setjmp.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/syslog.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/errno.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/inet.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/new.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/signal.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/threads.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/fcntl.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/intrin.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/noop.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/stdio.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/time.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/fenv.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/java.io.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/process.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/stdlib.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/unistd.c
+)
+
 add_custom_command(OUTPUT converter_input.txt
-    COMMAND cat ${CMAKE_CURRENT_SOURCE_DIR}/library/*.c > converter_input.txt
+    COMMAND cat ${ansi_c_library_sources} > converter_input.txt
+    DEPENDS ${ansi_c_library_sources}
 )
 
 add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc
-    COMMAND converter < converter_input.txt > ${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc
+    COMMAND converter < "converter_input.txt" > "${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc"
```

Comment:
Any guidance on when to quote/when not to quote?

Reply:
Much like a shell script, we quote when a path may contain spaces. The quotes around `converter_input.txt` aren't necessary, they're just for consistency

Reply:
Ok, thanks! I was indeed wondering about the ones around `converter_input.txt`, but also the fact that quoting was only used after `COMMAND` but not after `OUTPUT`.